openapi/paths/storefront/payment-instruments@{id}@setup.yaml

```
@@ -32,17 +32,17 @@ post:
     - Storefront
   tags:
     - Payment instruments
-  summary: Create payment instrument setup transaction
+  summary: Create a set up payment instrument transaction
```

As Adam mentioned, `setup` is the transaction type.
```suggestion
  summary: Create a setup payment instrument transaction
```